validator/validators/stress/stress_validator.go

```diff
@@ -188,14 +229,16 @@ func (s *StressValidator) ValidateStressMetric(metricName, metricNamespace strin
 	// Validate if the corresponding metrics are within the acceptable range [acceptable value +- 30%]
 	metricValue := metrics.MetricDataResults[0].Values[0]
 	upperBoundValue := metricPluginBoundValue[dataRate][receiver][metricName] * (1 + metricErrorBound)
+	log.Printf("Metric %s within the namespace %s has value of %f and the upper bound is %f", metricName, metricNamespace, metricValue, upperBoundValue)
```

Comment:
Would leaving this in make logs too verbose?

Reply:
That's should be fine. It would be good for the test owner to record the metrics value and average to know better the results and better to know if its within the bound of stress testing for that specific plugin or for that specific data rate.